File: Java/Sum.java

import java.util.Scanner;

public class Sum
{
    public static void main(String ar[])
	{
        Scanner s=new Scanner(System.in);
		System.out.println("Enter two numbers: ");
        int a=s.nextInt();
        int b=s.nextInt();
        int sum=a+b;
        System.out.println("Result: "+sum);
        s.close();
	}
}

File: Flutter/dart/bin/sum.dart
import 'dart:io';
void main()
{
  print('Enter two numbers');
  var input1 = stdin.readLineSync();
  var input2 = stdin.readLineSync();

  var num1 = int.parse(input1!);
  var num2 = int.parse(input2!);

  print('Sum = ${num1+ num2}');
}